operator/pkg/controller/dataset/controller.go

```diff
@@ -188,7 +188,7 @@ func (r *ReconcileDataset) reconcilePods(instance *api.EphemeralDataset) (bool,
 		reqLogger.Info("Making a new pod for dataset", "instance.Namespace", instance.Namespace, "instance.Name", instance.Name)
 		pod := r.newPod(instance)
 		if pod == nil {
-			reqLogger.Info("Pod was not created. Nfs is not up (yet)! Requeue after waiting for 10 seconds\n")
+			reqLogger.Info("Pod was not created. Nfs is not up yet or Benchmark didnt set its status fields yet or failed or failed to retrieve the Benchmark ! Requeue after waiting for 10 seconds\n")
```

Comment:
typo: didn't
This message is a little bit confusing. Please rephrase for better understating.